build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/Jacoco.kt

```diff
@@ -36,6 +37,10 @@ private val coverageExclusions = listOf(
     "**/Manifest*.*"
 )
 
+private fun String.capitalize() = replaceFirstChar {
```

Comment:
Is this needed? It doesn't seem to be used in this change.

Reply:
This fixes the following warning message:

```
w: file:///[...]/nowinandroid/build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/Jacoco.kt:51:48 'capitalize(): String' is deprecated. Use replaceFirstChar instead.
```

It appear unused but in fact, it is because the deprecated `String.capitalize()` extension is already defined in `kotlin.text` and does not need import statement.

Reply:
I see, makes sense.